src/fetch/orderbook.py

```diff
@@ -15,7 +15,7 @@
 from src.models.block_range import BlockRange
 from src.utils import open_query
 
-REORG_THRESHOLD = 65
+MAX_PROCESSING_DELAY = 1
```

Comment:
```suggestion
MAX_PROCESSING_DELAY = 10
```
I would increase the delay a bit just to be on the safe side.